.github/workflows/lint.yml

```diff
@@ -0,0 +1,21 @@
+name: Lint rules
+on: [pull_request]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: ricardochaves/python-lint@v1.1.0
+        with:
+          python-root-list: "python_fluentd_testing tests"
+          use-black: true
+          use-mypy: true
+          use-isort: true
+          use-pycodestyle: false
+          use-pylint: false
+          use-flake8: false
+          extra-black-options: "--diff"
+          extra-pycodestyle-options: ""
+          extra-flake8-options: ""
+          extra-mypy-options: ""
```

Comment:
you dont need "" value here

Reply:
@ricardochaves that's good to know. I copied from a project of ours. I'll change it.